src/main/java/io/vertx/ext/dropwizard/impl/VertxMetricsImpl.java

```diff
@@ -52,6 +48,8 @@
  * @author <a href="mailto:nscavell@redhat.com">Nick Scavelli</a>
  */
 class VertxMetricsImpl extends AbstractMetrics implements VertxMetrics {
+  private static final ThreadMXBean threadMXBean = ManagementFactory.getThreadMXBean();
+  private static int eventLoopThreads;
```

Comment:
As threadMXBean.getAllThreadIds is on the static instance returned by the runtime, I don't think there would be a change.